### ISSUE TITLE
Cannot download sources

### ISSUE THREAD

**Issue by mallowigi** [JetBrains/gradle-intellij-plugin#626]
Since the release of 2021.1 I am not able to download the sources anymore.

Error: 
```
Caused by: org.gradle.internal.resolve.ArtifactNotFoundException: Could not find ideaIC-LATEST-EAP-SNAPSHOT-sources.jar (com.jetbrains.intellij.idea:ideaIC:LATEST-EAP-SNAPSHOT).
```

**Comment by gharrma** [comment]
I ran into a similar issue as well when using `intellij.version = "2021.1"`.
I was able to work around it by specifying a more specific IJ version, like this:
```
intellij {
    version = "211.6693.111"
}
```
(I'm not sure why this fixed it for me.)

**Comment by mallowigi** [comment]
Yes, this will work, but I wonder why `LATEST-EAP-SNAPSHOT` is not available.

**Comment by YannCebron** [comment]
after release, EAP is closed, what kind of "latest EAP" do you expect?

**Comment by mallowigi** [comment]
As far as i remember that was never an issue before.

**Comment by henryju** [comment]
In our QA, we have an axis testing `LATEST-EAP-SNAPSHOT` to detect issues as soon as possible. Would be great to not have to change configuration every time there is a release. Maybe this is more a feature request, but would it be possible to have a `LATEST-BUILD` that would be either latest EAP, or latest release if no EAP are available?

**Comment by YannCebron** [comment]
the problem should be fixed with publication of next 2021.1 EAP release (very soon), please reopen if still reproducable
